### PR TITLE
Add column level string escaping

### PR DESCRIPTION
**Pull request by jackjamieson**
https://github.com/wenzhixin/bootstrap-table/issues/1049

Issue 1049 was resolved by adding a table-wide escape option.  This pull request adds the ability to escape on the column level so certain columns can still have markup in them with the `data-escape` attribute on the `<th>` element.

### CODE REVIEW

**Comment by djhvscf** [comment]
Please provide a [jsFiddle](https://github.com/wenzhixin/bootstrap-table/blob/develop/CONTRIBUTING.md#bug-reports) to show your problem, thanks!

**Comment by jackjamieson** [comment]
https://jsfiddle.net/knd8vff1/ this shows the current table-escaping.
https://jsfiddle.net/2esvsz26/1/ this shows the proposed table-level escaping addition.

**Comment by djhvscf** [comment]
Ok, got it... but please update the documentation

**Comment by jackjamieson** [comment]
Documentation updated.

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -829,6 +830,10 @@
                     that.header.stateField = column.field;
                     that.options.singleSelect = true;
                 }
+                
+                if(column.escape) {
+                	that.options.escapeColumn = true;
```
Comment (djhvscf) [inline src/bootstrap-table.js]: What about this option `escapeColumn`?

Comment (jackjamieson) [inline src/bootstrap-table.js]: Woops, good catch that was from earlier while I was testing it out.  I will remove it.

**Comment by jackjamieson** [comment]
Do I need to include an example within the documentation?

**Comment by wenzhixin** [comment]
Thanks for your contribution!

**Pull request merged**